README.md

```diff
@@ -23,14 +47,44 @@ Start postgres daemon:
 - macOS: `brew services start postgresql`
 - linux: `sudo service postgresql-13.2 start`
 
-Next set up the development and test databases. Make sure `npx` is working first (`npx --version`).
+Next set up the development and test databases. (Make
+sure `npx` is working first by running `npx --version`.)
 
 - `./script/init_db`
 
-If these command fail due to `could not connect to server: No such file or directory`, check the log file at `/usr/local/var/log/postgres.log`. If you see an error that looks similar to `The data directory was initialized by PostgreSQL version 11, which is not compatible with this version 13.2`, you have a previous version of postgresql installed that is conflicting with this one. The easiest way to resolve this is to `mv` the old data directory and run `initdb` with the new version of postgres. See https://gist.github.com/olivierlacan/e1bf5c34bc9f82e06bc0 for more details.
-
 Run the dev server:
 
-`npm run dev`
+`yarn dev`
 
 And then view graphql sandbox at http://localhost:3000/graphql
+
+If you run into problems setting up your development environment please create an issue describing any errors you encounter.
+
+See the README in the `frontend` directory for instructions on setting up for front end development.
+
+## Technologies
+
+Shipment Tracker is a full stack TypeScript web app backed by a PostgreSQL database.
+
+General tools:
+
+- [TypeScript](https://www.typescriptlang.org/)
+- [Yarn package manager](https://yarnpkg.com/)
+- [GraphQL](https://graphql.org/)
+- [GraphQL Codegen](https://graphql-code-generator.com/)
+- [Prettier code formatter](https://prettier.io/)
+- [Jest test framework](https://jestjs.io/)
+
+Back end:
+
+- [Node.js](https://nodejs.org/en/)
+- [PostgreSQL](https://www.postgresql.org/)
+- [Sequelize ORM](https://sequelize.org/)
+- [Apollo Server](https://www.apollographql.com/docs/apollo-server/)
+
+Front end:
+
+- [React.js](https://reactjs.org/)
+- [Next.js](https://nextjs.org/)
```

Comment:
Chore **(blocking)**: I ended up not using Next for various reasons, so you can remove that line.

**Suggestion (non-blocking)**: you could add React router here:
```md
- [React Router](https://reactrouter.com/web/guides/quick-start)
```